dist/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard App</title>
+  <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+  <body>
+    <section class="leaderboard-section">
+      <h1>Leaderboard</h1>
+      <div class="leaderboard-content-container">
+        <div class="leaderboard-left-section">
+          <div class="leaderboard-left-section-title">
+            <h2>Recent scores</h2>
+            <button type="button" class="refresh-btn">Refresh</button>
+          </div>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Implement%20the%20%22Refresh%22%20button%20that%20will%20get%20all%20scores%20for%20a%20game%20created%20by%20you%20from%20the%20API), Please make the app get the scores data when the refresh button clicked because now clicking it doesn't do anything.